docs/source/install_instructions.rst

```diff
@@ -12,10 +12,55 @@ OOIPY is available on PyPI.
 
 Install Instruction
 -------------------
-OOIPY is designed to run on Python 3.8.5. To install OOIPY, run the following command.
+OOIPY is designed to run on Python > 3.7 To install OOIPY, run the following command.
 
 .. code-block :: bash
 
   pip install ooipy
 
 Please see our Getting Started Guide for a tutorial on basic OOIPY usage.
+
+
+How to setup OOIPY development environment
+==========================================
+
+1. Navigate into the folder where you wish to setup ooipy and clone the repository using this command:
+
+.. code-block :: bash
+
+  git clone https://github.com/Ocean-Data-Lab/ooipy.git
+
+2. Setup a new Conda environment using this command:
+
+.. code-block :: bash
+
+  conda create --name env_name pip
```

Comment:
```suggestion
  conda create --name env_name python=3.10 pip
```

Let's specify the python version here, so user doesn't just end up installing the python version that matches their conda base, in case someone has an older conda python version. The above specifies python 3.10 just to be safe.